pkg/operator/operator.go

```diff
@@ -25,7 +25,9 @@ var (
 	setupLog = log.Log.WithName("operator")
 )
 
-func Run(buildInfo starboard.BuildInfo, operatorConfig etc.Config) error {
+// Start starts all registered reconcilers and blocks until the context is cancelled.
+// Returns an error if there is an error starting any reconciler.
+func Start(ctx context.Context, buildInfo starboard.BuildInfo, operatorConfig etc.Config) error {
```

Comment:
Adding context to programmatically stop the operator in integration tests. For example, to test with the default Polaris plugin, and then restart the operator with the Conftest plugin configured.
In production the operator is stopped on sig interrupt